AEPMedia/Sources/MediaPublicTracker.swift

```diff
@@ -160,25 +160,31 @@ class MediaPublicTracker: MediaTracker {
 
             let currentTs = self.getCurrentTimeStamp()
             if (currentTs - self.lastEventTs) > self.EVENT_TIMEOUT_MS {
-                // We have not got any public api call for 500 ms.
+                // We have not got any public api call for 1 second.
                 // We manually send an event to keep our internal processsing alive (idle tracking / ping processing).
                 self.trackInternal(eventName: MediaConstants.EventName.PLAYHEAD_UPDATE, params: self.lastPlayheadParams, internalEvent: true)
             }
         }
     }
 
     private func startTimer() {
-        if timer == nil {
-            timer = Timer.scheduledTimer(withTimeInterval: TICK_INTERVAL, repeats: true, block: { (_) in
-                self.tick()
-            })
-            timer?.fire()
+        guard self.timer == nil else {
```

Comment:
Use `if timer == nil { ... }` if just checking the value without unwrapping the variable.